scripts/import/import_EPMC.pl

```diff
@@ -993,9 +1017,199 @@ sub get_current_citations{
     return \%citations;
 }
 
+###
+# Get citations from phenotype tables
+
+sub process_phenotype_feature {
+  my $dba = shift;
+  my $source_ad = shift;
+  my $citation_attribs = shift;
+
+  my %list_citations_pheno_feature;
+
+  ## Get citations from phenotype_feature
+  my $pheno_citations_sth = $dba->dbc()->prepare(qq[ select s.study_id, s.source_id, s.external_reference, s.study_type, p.object_id
+                                                from study s
+                                                inner join phenotype_feature p on s.study_id = p.study_id
+                                                where p.type = 'variation' and p.study_id is not null and s.external_reference is not null
+                                                group by s.study_id, s.source_id, s.external_reference, s.study_type, p.object_id ]);
+  $pheno_citations_sth->execute()||die;
+  my $data = $pheno_citations_sth->fetchall_arrayref();
+
+  foreach my $l (@{$data}){
+    my $study_id = $l->[0];
+    my $source_id = $l->[1];
+    my $external_reference = $l->[2];
+    $external_reference =~ s/PMID://;
+    my $study_type = $l->[3];
+    my $rsid = $l->[4];
+
+    # Get attrib id for source - some are null 
+    my $source_attrib_id;
+    if(defined $study_type){
+      $source_attrib_id = $citation_attribs->{$study_type};
```

Comment:
It would be simpler to take source from the phenotype table. There is nothing to stop different sources reporting the same type of study.